AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin.swift

```diff
@@ -22,6 +23,21 @@ final public class AWSDataStorePlugin: DataStoreCategoryPlugin {
     /// The local storage provider. Resolved during configuration phase
     var storageEngine: StorageEngineBehavior!
 
+    var iCancellable: Any?
+
+    @available(iOS 13.0, *)
+    var cancellable: AnyCancellable? {
+        get {
+            if let iCancellable = iCancellable {
+                return (iCancellable as! AnyCancellable) //swiftlint:disable:this force_cast
+            }
+            return nil
+        }
```

Comment:
Avoid the force cast?
```
if let iCancellable = iCancellable as? AnyCancellable {
                return iCancellable
 }
return nil
```

Reply:
I thought i tried all combinations of this to make this work without a force cast -- but i guess not, thanks!